test/unit/helpers/FileSerializerMigrationHelperTest.php

```diff
@@ -0,0 +1,220 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ */
+
+namespace oat\generis\test\unit\helpers;
+
+use common_Config;
+use common_session_SessionManager;
+use common_test_TestUserSession;
+use core_kernel_persistence_smoothsql_SmoothModel;
+use oat\generis\Helper\FileSerializerMigrationHelper;
+use oat\generis\model\fileReference\ResourceFileSerializer;
+use oat\generis\model\fileReference\UrlFileSerializer;
+use oat\generis\model\GenerisRdf;
+use oat\generis\test\GenerisTestCase;
+use oat\oatbox\filesystem\Directory;
+use oat\oatbox\filesystem\File;
+use oat\oatbox\filesystem\FileSystemService;
+use oat\oatbox\filesystem\FileSystemHandler;
+
+/**
+ * Test cases for the File serializer migration script helper
+ * @see \oat\generis\Helper\FileSerializerMigrationHelper
+ */
+class FileSerializerMigrationHelperTest extends GenerisTestCase
+{
+
+    const PARENT_RESOURCE_URI = 'http://www.tao.lu/Ontologies/generis.rdf#UnitTest';
+    const PROPERTY_URI = 'http://www.tao.lu/Ontologies/generis.rdf#TestFile';
+
+    /**
+     * @var FileSerializerMigrationHelper
+     */
+    protected $fileMigrationHelper;
+
+    /**
+     * @var \core_kernel_classes_Class
+     */
+    protected $testClass;
+
+    /**
+     * @var core_kernel_persistence_smoothsql_SmoothModel
+     */
+    private $ontologyMock;
+
+    /**
+     * @var ResourceFileSerializer
+     */
+    private $resourceFileSerializer;
+
+    /**
+     * @var UrlFileSerializer
+     */
+    private $urlFileSerializer;
+
+    /**
+     * @var string
+     */
+    private $tempFileSystemId;
+
+    /**
+     * @var
+     */
+    private $tempDirectory;
+
+    /**
+     * @var File|Directory
+     */
+    private $testFile;
+
+    /**
+     * @var FileSystemService
+     */
+    private $fileSystemService;
+
+    /**
+     * Initialize test
+     */
+    public function setUp()
+    {
+        common_Config::load();
```

Comment:
Why would you need to load the config of the current install in a unit test?

Reply:
up